### ISSUE TITLE
Adding functx unnecessary?

### ISSUE THREAD

**Issue by bwbohl** [peterstadler/existdb-docker#2]
https://github.com/peterstadler/existdb-docker/blob/7345e489737df8b07fefc267552e47c3f00ea4b1/Dockerfile#L47

Adding the functx-1.0.1.xar is unnecessary, at lest for eXist-db v5.2.0, as it's already included in the autodeploy directory. Is there any reason for keeping this, e.g. for older eXist versions?

**Comment by peterstadler** [comment]
Yes, you are right. This looks like superfluous legacy code. Do you want to remove it as you're working on #3 ?
